template/.eslintrc.js

```diff
@@ -10,132 +10,132 @@ module.exports = {
-        }
-      }
-    ],
-    "@typescript-eslint/member-ordering": "error",
-    "@typescript-eslint/no-empty-function": "error",
-    "@typescript-eslint/no-empty-interface": "error",
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-misused-new": "error",
-    "@typescript-eslint/no-namespace": "error",
-    "@typescript-eslint/no-parameter-properties": "off",
-    "@typescript-eslint/no-use-before-define": "off",
-    "@typescript-eslint/no-var-requires": "error",
-    "@typescript-eslint/prefer-for-of": "error",
-    "@typescript-eslint/prefer-function-type": "error",
-    "@typescript-eslint/prefer-namespace-keyword": "error",
-    "@typescript-eslint/quotes": [
-      "error",
-      "double",
-      {
-        avoidEscape: true
-      }
-    ],
-    "@typescript-eslint/semi": ["error", "always"],
-    "@typescript-eslint/triple-slash-reference": "error",
-    "@typescript-eslint/type-annotation-spacing": "error",
-    "@typescript-eslint/unified-signatures": "error",
-    "arrow-body-style": "error",
-    "arrow-parens": ["error", "as-needed"],
-    camelcase: "error",
-    "capitalized-comments": "error",
-    "comma-dangle": ["error", "always-multiline"],
-    complexity: "off",
-    "constructor-super": "error",
-    curly: "error",
-    "dot-notation": "error",
-    "eol-last": "error",
-    eqeqeq: ["error", "smart"],
-    "guard-for-in": "error",
-    "id-blacklist": ["error", "any", "Number", "number", "String", "string", "Boolean", "boolean", "Undefined", "undefined"],
-    "id-match": "error",
-    "import/order": "error",
-    "max-classes-per-file": ["error", 1],
-    "max-len": [
-      "error",
-      {
-        code: 120
-      }
-    ],
-    "new-parens": "error",
-    "no-bitwise": "error",
-    "no-caller": "error",
-    "no-cond-assign": "error",
-    "no-console": "off",
-    "no-debugger": "error",
-    "no-empty": "error",
-    "no-eval": "error",
-    "no-fallthrough": "off",
-    "no-invalid-this": "off",
-    "no-multiple-empty-lines": "error",
-    "no-new-wrappers": "error",
-    "no-shadow": [
-      "error",
-      {
-        hoist: "all"
-      }
-    ],
-    "no-throw-literal": "error",
-    "no-trailing-spaces": "error",
-    "no-undef-init": "error",
-    "no-underscore-dangle": "error",
-    "no-unsafe-finally": "error",
-    "no-unused-expressions": "error",
-    "no-unused-labels": "error",
-    "no-var": "error",
-    "object-shorthand": "error",
-    "one-var": ["error", "never"],
-    "prefer-arrow/prefer-arrow-functions": "error",
-    "prefer-const": "error",
-    "quote-props": ["error", "consistent-as-needed"],
-    radix: "error",
-    "space-before-function-paren": [
-      "error",
-      {
-        anonymous: "never",
-        asyncArrow: "always",
-        named: "never"
-      }
-    ],
-    "spaced-comment": "error",
-    "use-isnan": "error",
-    "valid-typeof": "off",
-  }
+  // rules: {
+  //   "@typescript-eslint/adjacent-overload-signatures": "error",
```

Comment:
Why skipped all rules?

Reply:
because some rules are not followed in the core as class-name-casing, so it is more interesting for the user to be able to put the rules that he finds interesting to use